examples/cmake_installed/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.18.0)
+set(example pressio-example)
+project(${example})
+
+find_package(pressio REQUIRED)
+
+add_executable(${example} ${CMAKE_CURRENT_SOURCE_DIR}/../main.cpp)
+target_link_libraries(${example} PUBLIC pressio)
+target_compile_definitions(${example} PUBLIC PRESSIO_ENABLE_TPL_EIGEN)
```

Comment:
Since we are dropping `cmake/pressioConfig.cmake.in`, the configuration has to be done explicitly by the application that is using `pressio`. This means at least defining `PRESSIO_ENABLE_TPL_EIGEN`.